.github/workflows/ci.yml

```diff
@@ -57,35 +60,54 @@ jobs:
           echo "  github.event.ref: ${{ github.event.ref }}"
           echo "---"
           echo "concurrency-group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}"
+      - name: labeler
+        id: labeler
+        if: github.event_name == 'pull_request'
+        uses: lablup/auto-labeler@main   # actions/labeler, lablup/size-label-action, lablup/auto-label-in-issue
       - name: Optimize CI
-        id: check_skip
+        id: check_stacked_pr
         run: |
-          echo "SKIP=false" >> $GITHUB_OUTPUT
+          echo "NON_TOP_BOTTOM=false" >> $GITHUB_OUTPUT
```

Comment:
What does `NON_TOP_BOTTOM` mean?

Reply:
It means that among the stacked PRs, it is not the top or bottom stack.